app/templates/components/repo-status-badge.hbs

```diff
@@ -0,0 +1,11 @@
+<div class="repo-badge inline-block vertical-align">
+  <a
+    title="Latest push build on default branch: {{this.lastBuildState}}"
+    name="status-images"
+    class="pointer open-popup"
+    data-test-status-image-popup="true"
+    {{on 'click' this.onClick}}
```

Comment:
As long as it's an external attribute, what do you think about
```suggestion
    {{on 'click' @onClick}}
```
?

Reply:
Totally, thanks for catching it!